src/plot.jl

```diff
@@ -195,37 +195,37 @@ function _compute_plot_data(
     i::Integer,
     chains::Chains,
     par_names::AbstractVector{Symbol};
-    hpd_val = [0.05, 0.2],
+    hdi_prob = [0.94, 0.8],
     q = [0.1, 0.9],
     spacer = 0.4,
     _riser = 0.2,
     barbounds = (-Inf, Inf),
     show_mean = true,
     show_median = true,
     show_qi = false,
-    show_hpdi = true,
+    show_hdii = true,
     fill_q = true,
-    fill_hpd = false,
+    fill_hdi = false,
     ordered = false
 )
 
-    chain_dic = Dict(zip(quantile(chains)[:,1], quantile(chains)[:,4]))
+    chain_dic = Dict(zip(quantile(chains)[2], quantile(chains)[5]))
```

Comment:
Maybe a good opportunity to make the code a bit more efficient:
```suggestion
    quantile_chains = quantile(chains)
    chain_dic = Dict(zip(quantile_chains[2], quantile_chains[5]))
```
(it would also be nice if it would be possible to use something more descriptive than `2` and `5`)

Reply:
Hmm, yeah this and the code immediately below are a bit of a mess. Will rework to use the new methods.

Reply:
Yeah it was able to be simplified a lot. But also, for n parameters, it was computing the median n^2 times. This has now been fixed.